accounts/tests.py

```diff
@@ -169,78 +173,177 @@ def test_failure_post_with_password_similar_to_username(self):
+        self.assertTemplateUsed(response_get, 'accounts/login.html')
 
     def test_success_post(self):
-        pass
+        
+        data_post = {
+            'username':'yamada',
+            'password':'wasurenaide1108',
+        }
+       
+        
+        response_post = self.client.post(self.url, data_post)
+        
+        self.assertRedirects(response_post, reverse(settings.LOGIN_REDIRECT_URL), status_code=302, target_status_code=200)
+        self.assertIn(SESSION_KEY, self.client.session)
 
+       
+        
     def test_failure_post_with_not_exists_user(self):
-        pass
+        #存在しないusername,passwordを送信する
+        data_not_exists_user = {
+            'username': 'aaaaaaaaaa',
+            'password': 'Hasse118',
+        }
+        response_not_exists_user = self.client.post(self.url, data_not_exists_user)
+        self.assertEquals(response_not_exists_user.status_code, 200)
+        self.assertFormError(response_not_exists_user, 'form', '', '正しいユーザー名とパスワードを入力してください。どちらのフィールドも大文字と小文字は区別されます。')
+        self.assertNotIn(SESSION_KEY, self.client.session)
 
     def test_failure_post_with_empty_password(self):
-        pass
+        data_with_empty_password = {
+            'username' : '長谷川滉大',
+            'password' : '',
+        }
+
+        response_with_empty_password=self.client.post(self.url, data_with_empty_password)
+        self.assertEquals(response_with_empty_password.status_code, 200)
+        self.assertFormError(response_with_empty_password, 'form', 'password', 'このフィールドは必須です。')
+        self.assertNotIn(SESSION_KEY, self.client.session)
 
 
 class TestLogoutView(TestCase):
-    def test_success_get(self):
-        pass
+
+    def setUp(self):
+        data = {
+            'username': 'yamada',
+            'email': 'asaka@test.com',
+            'password1': 'wasurenaide1108',
+            'password2': 'wasurenaide1108',
+        }
+        self.client.post(reverse('accounts:signup'), data)
+
+    def test_success_logout(self):
+        response = self.client.get(reverse('accounts:logout'))
+        
+        self.assertRedirects(response, reverse(settings.LOGOUT_REDIRECT_URL), status_code=302, target_status_code=200)
+        self.assertNotIn(SESSION_KEY, self.client.session)
 
 
 class TestUserProfileView(TestCase):
+    def setUp(self):
+        User.objects.create_user(username='yamada', email='asaka@test.com', password='wasurenaide1108')
+        self.client.login(username='yamada', password='wasurenaide1108')
+      
+
     def test_success_get(self):
-        pass
+        user = User.objects.get(username='yamada')
+        response_get = self.client.get(reverse('accounts:user_profile', kwargs={'pk':user.pk}))
+        self.assertEqual(response_get.status_code, 200)
+        self.assertTemplateUsed(response_get, 'accounts/profile.html')
+
+    def test_failure_get_with_not_exists_user(self):
+        response = self.client.get(reverse('accounts:user_profile', args=[1000]))

```

Comment:
```suggestion
        response = self.client.get(reverse('accounts:user_profile', kwargs={'pk':1000}))
```
のように形式は統一したほうがわかりやすいと思います。